components/script/dom/htmlscriptelement.rs

```diff
@@ -78,6 +81,7 @@ impl HTMLScriptElement {
             ready_to_be_parser_executed: Cell::new(false),
```

Comment:
The checks for ParserCeated on the previous line (and elsewhere in this PR) are still incorrect.